h/api/models/elastic.py

```diff
@@ -117,7 +117,7 @@ def shared(self):
         elif len(read_perms) > 1 and self.groupid == '__world__' and gperm in read_perms:
             read_perms = [gperm]
 
-        if (read_perms != [gperm] and read_perms != [self.userid]):
+        if (gperm not in read_perms and read_perms != [self.userid]):
```

Comment:
What's the motivation for this change? This entire function is intended to be very conservative in its interpretation of the permissions field, precisely so we can be sure we're importing correct data into PostgreSQL.

If there are annotations where `read_perms` hasn't been successfully mapped to a single-element list by one of the clauses immediately above this line, that's a problem, and we shouldn't fix it by relaxing expectations here.